src/main/java/uk/gov/hmcts/reform/managecase/client/datastore/CaseDetails.java

```diff
@@ -33,6 +33,7 @@
 public class CaseDetails {
 
     public static final String ORGANISATION_TO_ADD = "OrganisationToAdd";
+    public static final String CREATED_BY = "CreatedBy";
```

Comment:
We should probably add `LastNoCRequestedBy` as a constant here too.

Reply:
done